### ISSUE TITLE
How well did it work?

### ISSUE THREAD

**Issue by patientzero** [thomasschmied/Speech_Recognition_with_Tensorflow#11]
Hi Thomas,

since I cant find your Email anywhere I decided to write you here. 
First of: Thanks for publishing the code. 
Second: I extended your code a little and ran a couple of experiments with it, but the decdoding never gets as good as your's was, when overfitting the model.
First: Training only on dev set, with early stopping: not really good, but the direction was about right.(decoded test/dev set with it)
Second: Overfitting on the dev set for 500 epochs without early stopping: Did not lead to improvements over first try. 
Third: Training the model on the Trainset(train-clean, 100h). After about 100 Epochs decoding results for test/dev set are still worse than first/second. Experiment is still running and I save a model every now and then. 

What am I doing wrong? And do you by any chance still have the model that led to the good decoding results shown in the notebook? 
Thanks, Sebastian

**Comment by rohith3679** [comment]
hi @patientzero i have some issues while running this code can you please help me to sort them out.

**Comment by patientzero** [comment]
I think you must not end the path with a '/' that should fix the problem. 
Familiarize yourself with the old checkpoint format that tensorflow used.

**Comment by Mohammadelc** [comment]
> Hi Thomas,
> 
> since I cant find your Email anywhere I decided to write you here.
> First of: Thanks for publishing the code.
> Second: I extended your code a little and ran a couple of experiments with it, but the decdoding never gets as good as your's was, when overfitting the model.
> First: Training only on dev set, with early stopping: not really good, but the direction was about right.(decoded test/dev set with it)
> Second: Overfitting on the dev set for 500 epochs without early stopping: Did not lead to improvements over first try.
> Third: Training the model on the Trainset(train-clean, 100h). After about 100 Epochs decoding results for test/dev set are still worse than first/second. Experiment is still running and I save a model every now and then.
> 
> What am I doing wrong? And do you by any chance still have the model that led to the good decoding results shown in the notebook?
> Thanks, Sebastian

Hi Sebastian, 
I am trying to rerun the codes but got the similar problem. And I couldn't get the result posted in the jupyter notebook. The lowest loss that I have gotten so far is about 1.5 which is more than the lost (0.4) that the author achieved. Have you solved this problem and got the same result? Or do you have any suggestion that could help me to get rid of this problem?
Thank you,
Mohammad

**Comment by patientzero** [comment]
Hi @Mohammadelc, 
I gave up on trying. I couldn't make it work properly that was why I was asking in the first place. I even tried different data, but somehow had difficulties to make in converge. 
I think something with the general implementation is off, but as I needed it back then, I moved to as simpler task and a simple key word recognizer  which was enough to prove what we were trying to do.   
Cheers,
Sebastian

**Comment by Mohammadelc** [comment]
Thank you @patientzero for replying,
I have been working on this code for two weeks, the result for the training set is reasonable but for a test is totally very bad. I should say that it is random and there is no trace of learning on a test set. Also, I have tried the codes on two datasets but the findings are the same. 

Mohammad

**Comment by masoudMZB** [comment]
@Mohammadelc  can I know more about your experiment with this repo?
I am using this repo to understand how to use attention and beam. 
thanks

**Comment by Mohammadelc** [comment]
@masoudMZB 
I was working on it for two types of dataset but unfortunately couldn't get any results reasonable. The results in the test phase are so random for a Character-based ASR model at least. 

Hope it helps,
